scripts/travis_install.sh

```diff
@@ -6,7 +6,6 @@ set -ev
 set -o pipefail
 
 # Building should work without yarn installed globally
-rm /home/travis/.yarn/bin/*
 !yarn --version
```

Comment:
I think we need the `rm` above - it was specifically testing the case where yarn is not installed globally, fixing an issue we've had before.